benchmarking.md

```diff
@@ -44,33 +44,35 @@ This produces large volumes of Kafka logs in _/tmp/kafka-logs_ which should remo
 
 ## Results
 
-Results from a benchmark run on MacMini M1.
+Results from a benchmark run on Lenovo T14s.
```

Comment:
We should probably include the exact configuration used when obtaining the results to make it easier for other to validate our benchmarks.

E.g. JVM version & distribution, and kafka props and Kroxylicious config.